crates/astria-sequencer/src/accounts/action.rs

```diff
@@ -73,7 +76,10 @@ impl ActionHandler for Transfer {
             .get_account_balance(from)
             .await
             .context("failed getting `from` account balance")?;
-        ensure!(curr_balance >= self.amount, "insufficient funds");
+        ensure!(
```

Comment:
Similar to other comments I left - I think it would be nice to start using more typed errors.